### ISSUE TITLE
Missing getFloat

### ISSUE THREAD

**Issue by sevarac** [lightbend/config#527]
Hi, 

Cool config lib, but Config interface is missing getFloat()
Will it be added?

Thanks

**Comment by ktoso** [comment]
What's wrong with doubles?

**Comment by sevarac** [comment]
Nothing wrong with doubles,its just I need float, and dont want to cast

**Comment by ktoso** [comment]
Hmm, I guess. Another ticket where I'd like to ask @havocp's opinion (when you have the time Havoc)

**Comment by havocp** [comment]
No strong view. Generally I think using float is broken and uncommon, with the exception of weird hardware-related reasons (like GPUs), so didn’t bother.

But maybe could be there for completeness especially if parseFloat somehow preserves more precision than parsing double and casting? If the implementation would just cast getDouble that doesn’t seem super useful.

**Comment by ktoso** [comment]
> But maybe could be there for completeness especially if parseFloat somehow preserves more precision than parsing double and casting?

Yeah, good point. Worth checking that then and basing the decision on that I guess then

**Comment by sevarac** [comment]
I think that it should be there  for completeness as you said. Since there is getInt and getLong.
Pls add :) I really like this lib, but I dont want to have to cast getDouble to float :(  Thanks!
